storage/database/src/main/java/de/danoeh/antennapod/storage/database/PodDBAdapter.java

```diff
@@ -1200,10 +1200,6 @@ public final int getNumberOfNewItems() {
     public final Map<Long, Integer> getFeedCounters(FeedCounter setting, long... feedIds) {
         String whereRead;
         switch (setting) {
-            case SHOW_NEW_UNPLAYED_SUM:
-                whereRead = "(" + KEY_READ + "=" + FeedItem.NEW +
-                        " OR " + KEY_READ + "=" + FeedItem.UNPLAYED + ")";
-                break;
```

Comment:
After this migration, the "unplayed" counter should take over the behavior of "new and unplayed", because episodes in the inbox are always unplayed. Our data model here is a bit confusing because an episode can be either in the inbox or unplayed, but for a user, episodes in the inbox are definitely unplayed as well.

Reply:
Thank you, I have updated this in my most recent commit.